CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+- For issues concerning versions of Business Central that are in mainstream support you should use the [support channel](#support).
+
+### Look for an existing issue
+
+Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/AL/issues) to see if the issue or feature request has already been filed.
+
+Be sure to scan through the [most popular](https://github.com/microsoft/AL/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+) issues.
+
+If you find that your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment:
+
+- 👍 - upvote
+- 👎 - downvote
+
+If you cannot find an existing issue that describes your bug, create a new issue using the guidelines below.
+
+### Writing good bug reports
+
+File a single issue per problem. Do not enumerate multiple bugs or feature requests in the same issue.
+
+Do not add your issue as a comment to an existing issue unless it's for the identical input. Many issues look similar, but have different causes.
+
+The more information you can provide, the more likely someone will be successful at reproducing the issue and finding a fix.
+
+Please include the following with each issue:
+
+**1. Describe the bug**
+A clear and concise description of what the bug is. 
+
+**2. To Reproduce**
+Steps and to reproduce the behavior:
+
+1. Go to '...'
+
+``` AL code snippet that demonstrates the issue or a link to a code repository the developers can easily pull down to recreate the issue locally ```
+
+**Note:** Because the developers need to copy and paste the code snippet, including a code snippet as a media file (i.e. .gif) is not sufficient.
+
+**3. Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**4. Actual behavior**
+A clear and concise description of what happened accompanied by images, animations, or a link to a video showing the issue occurring.
+
+**5. Versions:**
+
+- AL Language. Go to Visual Studio Code → Extensions panel → AL Language 
+- Visual Studio Code: Go to Visual Studio Code → Help → About
+- Business Central. In the client search for: System Information
+- List of extensions that you have installed
+
+### Final checklist
+
+Please remember to do the following:
+
+* [ ] Search the issue repository to ensure you are reporting a new issue
+
+* [ ] Reproduce the issue after switching off all extensions except the AL Language extension
+
+* [ ] Simplify your code around the issue to better isolate the problem
+
+### Follow your issue
+
+Once submitted, your report will go into the [issue tracking](https://github.com/microsoft/AL/wiki/Issue-Tracking) workflow. Be sure to understand what will happen next, so that you know what to expect, and how to continue to assist throughout the process.
```

Comment:
Sooo, what will happen? Where can I read more about this if I do not know? Generic GitHub issue tracking docs? E.g. https://guides.github.com/features/issues/

Reply:
FYI, just in case it's not intended/expected:
While we users have permission to view https://github.com/microsoft/AL/wiki, we do not have it to view https://github.com/microsoft/AL/wiki/Issue-Tracking.